server/src/main/java/org/elasticsearch/health/HealthFeatures.java

```diff
@@ -16,7 +16,7 @@
 
 public class HealthFeatures implements FeatureSpecification {
 
-    public static final NodeFeature SUPPORTS_HEALTH = new NodeFeature("supports_health");
+    public static final NodeFeature SUPPORTS_HEALTH = new NodeFeature("health.supports_health");
```

Comment:
Historical features are not actually published anywhere, so it's ok to change the id here

Reply:
Ah, thanks, TIL.